src/main.jsx

```diff
@@ -178,6 +178,14 @@ class Main extends Component {
           history = history.filter(m => m.optout);
         }
 
+        // Add a detailed type to differentiate between regular and keyed probes
+        if (history[0]["details"]["keyed"]) {
+          // Ensure that we only add the word "keyed" once, since React lifecycle can call this method multiple times
```

Comment:
I'm not sure why this is needed since we're adding the word "keyed" to the value of `type` and storing it in `detailed_type` where we only ever read it for display... there should be no possibility of doubling up, right?